api/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
-
+from django.utils import timezone
```

Comment:
I believe you do not need to import timezone. Had the same comment from Jure on projects model. The auto_now property implies that.